src/commands/app/deploy.js

```diff
@@ -94,8 +96,12 @@ class Deploy extends BuildCommand {
 
       // 3. send deploy log event
       const logEvent = getAuditLogEvent(flags, aioConfig.project, 'AB_APP_DEPLOY')
-      if (logEvent) {
-        await sendAuditLogs(cliDetails.accessToken, logEvent, cliDetails.env)
+      if (logEvent && cliDetails) {
+        try {
+          await sendAuditLogs(cliDetails.accessToken, logEvent, cliDetails.env)
+        } catch (error) {
+          this.log(chalk.red(chalk.bold('Error: Audit Log Service Error: Failed to send audit log event for deployment.')))
```

Comment:
it is an error in sending, but to the cli is it an error? I would say it's more like a warning here. Just do a `this.warn` without all the `chalk` adornment